public/javascripts/Progress/src/Progress.js

```diff
@@ -51,30 +51,57 @@ function Progress (_, $, userRole) {
         });
     }
 
-    function putUserOrg(e) {
+    function putUserTeam(e, newTeam) {
         var parsedId = $(this).attr('id').split("-"); // the id comes in the form of "from-startOrg-to-endOrg"
-        var startOrg = parsedId[1];
-        var endOrg = parsedId[3];
+        var startTeam = parsedId[1];
+        if (newTeam != null) {
```

Comment:
The main issue with this is that you're declaring `endTeam` in two different places, which isn't great. This would be written more simply as a one-liner:
```
var endTeam = newTeam ? newTeam : parseId[3];
```

And this has the benefit of making it so that you're not declaring `endTeam` twice!